packages/aws-client/src/S3.ts

```diff
@@ -1121,39 +1058,25 @@ export const multipartCopyObject = async (
 };
 
 /**
- * Move an S3 object to another location in S3
- *
- * @param {Object} params
- * @param {string} params.sourceBucket
- * @param {string} params.sourceKey
- * @param {string} params.destinationBucket
- * @param {string} params.destinationKey
- * @param {string} [params.ACL] - an [S3 Canned ACL](https://docs.aws.amazon.com/AmazonS3/latest/dev/acl-overview.html#canned-acl)
- * @param {boolean} [params.copyTags=false]
- * @param {number} [params.chunkSize] - chunk size of the S3 multipart uploads
- * @returns {Promise<undefined>}
+ * Copy an S3 object to another location in S3
  */
-export const moveObject = async (
-  params: {
-    sourceBucket: string,
-    sourceKey: string,
-    destinationBucket: string,
-    destinationKey: string,
-    ACL?: string,
-    copyTags?: boolean,
-    chunkSize?: number
-  }
-) => {
-  const {
-    sourceBucket,
-    sourceKey,
-    destinationBucket,
-    destinationKey,
-    ACL,
-    copyTags,
-    chunkSize,
-  } = params;
-
+export const copyObject = async ({
+  sourceBucket,
```

Comment:
I'm concerned we're losing some of the param comments in these docstring removals - we definitely shouldn't require *typing* here, but the loss of the ACL link feels ... meaningful.